### PR TITLE
Updates Multi-language Java examples documentation

### PR DESCRIPTION
**Pull request by chamikaramj**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by chamikaramj** [comment]
R: @ihji

File: examples/multi-language/README.md [hunk 1]
```diff
@@ -126,9 +126,25 @@ gsutil cat gs://$GCP_BUCKET/multi-language-beam/output*
...
+
+```
+pip install apache-beam[gcp]
+pip install sklearn
+```
+
+* Startup the expansion service
+
+```
+python -m apache_beam.runners.portability.expansion_service_main -p <PORT> --fully_qualified_name_glob "*"
```
Comment (ihji) [inline examples/multi-language/README.md:142]: Do we need --pickle_library=cloudpickle ?

File: website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md [hunk 2]
```diff
@@ -243,9 +243,9 @@ pip install apache-beam[gcp,dataframe]
 
 4. Run the following command
 
-   ```
-   python -m apache_beam.runners.portability.expansion_service_main -p <PORT> --fully_qualified_name_glob "*"
-   ```
+```
+python -m apache_beam.runners.portability.expansion_service_main -p <PORT> --fully_qualified_name_glob "*"
```
Comment (ihji) [inline website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md:247]: ditto

**Review by ihji — APPROVED** [review]
LGTM. Thanks!

File: examples/multi-language/README.md (hunk 1 above)
Comment (chamikaramj) [reply examples/multi-language/README.md:142]: I don't think we can add it till https://github.com/apache/beam/pull/23739 is released :(.

After that we can update all guides/examples.

File: website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md (hunk 2 above)
Comment (chamikaramj) [reply website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md:247]: Ditto.

**Pull request merged**